src/components/dashboard/Marketplace.js

```diff
@@ -84,16 +85,19 @@ const navBarStyles = {
   },
 }
 
-const NavigationBar = navigate => (
-  <Appbar.Header dark style={navBarStyles.wrapper}>
-    <Section.Text color="white" fontWeight="medium" style={navBarStyles.title}>
-      {'GOODMARKET'}
-    </Section.Text>
-    <TouchableOpacity onPress={() => navigate('Home')} style={navBarStyles.walletIcon}>
-      <Icon name="wallet" size={36} color="white" />
-    </TouchableOpacity>
-  </Appbar.Header>
-)
+const NavigationBar = navigate => {
```

Comment:
actually it's an functional component so let it be component:
```suggestion
const NavigationBar = ({ navigate }) => {
```

Reply:
and below:

```javascript
navigationBar: () => <NavigationBar navigate={navigation.navigate} />
```

Reply:
done